test/controllers/v1/profiles_controller_test.rb

```diff
@@ -888,7 +890,17 @@ class CreateTest < ProfilesControllerTest
 
       test 'create allows hosts relationship' do
         hosts = FactoryBot.create_list(:host, 2)
-        parent = FactoryBot.create(:canonical_profile)
+        parent = FactoryBot.create(:canonical_profile, upstream: false)
+        SupportedSsg.stubs(:raw_supported).returns(
+          'supported' => hosts.each_with_object({}) do |host, obj|
+            obj["RHEL-#{host.os_major_version}.#{host.os_minor_version}"] ||= []
+
+            obj["RHEL-#{host.os_major_version}.#{host.os_minor_version}"].push(
+              'version' => parent.benchmark.version
+            )
+          end
+        )
```

Comment:
@marleystipich2 you should mock supported SSG calls something like this in #1278, maybe even extract it into a helper method because it will be called at multiple places. This one specifically sets the supported versions based on two hosts to a given benchmark version, but it can be generalized.

Reply:
Okay, partially taking this back. Mocking `raw_supported` is a bad idea as it is not always being consumed due to multithreading and caching. So you should mock something higher, like I do with `by_ssg_version`. However, this method is specific to these tests, so it won't always work.